package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@volusion/element-cli",
-  "version": "1.0.0",
+  "version": "2.0.0",
```

Comment:
@bpeters We use a tool called `np` (check packages.json) that ask us what do we want to do with the version before releasing, and releases the package to npm. Maybe we should keep relying on that.

Reply:
oh okay, sorry I needed to bump locally to get BTR to work :) I'll remove from PR

Reply:
@bpeters Thanks!